dev_guide/deployments.adoc

```diff
@@ -10,27 +10,35 @@
 toc::[]
 
 == Overview
-A deployment in OpenShift is an update to a single replication controller's pod
-template based on triggered events. The `*deployment*` subsystem provides:
 
-- A link:#defining-a-deploymentConfig[deploymentconfig] that drives automated deployments by the system.
-- link:#triggers[Triggers] which drive new deployments in response to events.
-- User-customizable link:#strategies[strategies] for deployment rollout behavior, which are responsible for making a deployment live in the cluster.
+A deployment in OpenShift is a replication controller based on a user defined
+template called a deployment configuration. Deployments are created manually
+or in response to triggered events.
+
+The deployment system provides:
+
+- A link:#creating-a-deployment-configuration[deployment configuration], which is a template for deployments.
+- link:#triggers[Triggers] that drive automated deployments in response to events.
+- User-customizable link:#strategies[strategies] for deployment rollout behavior which are responsible for making a deployment active in the cluster.
```

Comment:
I'd make a mention of the fact that the "strategy" is also responsible for transitioning from the previous deployment to the new deployment.